mahta/CSS/CSS basics/style.css

```diff
@@ -0,0 +1,6 @@
+
```

Comment:
1. Removing redundant empty lines is highly preferred.
2. As I mentioned earlier, please do not include whitespace characters in file and directory names.

Reply:
I fixed everything I thought I should. I'm sorry for the problems you had to mention more than once and I'll try to pay better attention.